### PR TITLE
feat: display properly result fields of process

### PR DESCRIPTION
**Pull request by sebbousquet**
deps: use components v24.3.0-beta.7

### CODE REVIEW

File: projects/arlas-toolkit/assets/i18n/en.json [hunk 1]
```diff
@@ -14,6 +14,15 @@
 	"Count": "Number of elements",
 	"View Combine": "View Combine",
 	"Combine": "Combine",
+	"Change password": "Change password",
+	"Old password": "Old password",
+	"Password": "Password",
+	"Confirm password": "Confirm password",
+	"Password and Confirm Password must be match": "Password and Confirm Password must be match",
```
Comment (QuCMGisaia) [inline projects/arlas-toolkit/assets/i18n/en.json:21]: must be a match or must match

**Review by MohamedHamouGisaia — APPROVED** [review]

File: projects/arlas-toolkit/assets/i18n/fr.json [hunk 2]
```diff
@@ -1,247 +1,253 @@
...
+	"Name": "Nom",
+	"Add a dataset": "Ajouter un dataset",
+	"Manage datasets": "Gérer les datasets",
+	"Dataset name": "Nom du dataset",
+	"OK": "OK",
+	"Cancel": "Annuler",
+	"Count": "Nombre d'éléments",
+	"View Combine": "Simuler la combinaison",
+	"Combine": "Combiner",
+	"Change password": "Changement ",
```
Comment (QuCMGisaia) [inline projects/arlas-toolkit/assets/i18n/fr.json:17]: Changement du mot de passe

File: projects/arlas-toolkit/assets/i18n/fr.json [hunk 3]
```diff
@@ -1,247 +1,253 @@
...
+	"Old password": "Ancien mot de passe",
+	"Password": "Mot de passe",
+	"Confirm password": "Confirmation",
+	"Password and Confirm Password must be match": "Les champs Mot de passe et Confirmation doivent être identiques",
```
Comment (QuCMGisaia) [inline projects/arlas-toolkit/assets/i18n/fr.json:21]: I would quote the name of the fields

**Review by QuCMGisaia — APPROVED** [review]

File: projects/arlas-toolkit/assets/i18n/en.json (hunk 1 above)
Comment (sebbousquet) [reply projects/arlas-toolkit/assets/i18n/en.json]: ✅

File: projects/arlas-toolkit/assets/i18n/fr.json (hunk 2 above)
Comment (sebbousquet) [reply projects/arlas-toolkit/assets/i18n/fr.json]: ✅

File: projects/arlas-toolkit/assets/i18n/fr.json (hunk 3 above)
Comment (sebbousquet) [reply projects/arlas-toolkit/assets/i18n/fr.json]: ✅

**Pull request merged**